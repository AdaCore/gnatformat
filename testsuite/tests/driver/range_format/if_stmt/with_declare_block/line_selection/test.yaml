description: IfStmt formatting with partial selection.
driver: gnatformat
args:
  [
    "main.adb",
    "--range-format",
    "-SL 5",
    "-SC 7",
    "-EL 5",
    "-EC 17",
    "--pipe"
  ]
