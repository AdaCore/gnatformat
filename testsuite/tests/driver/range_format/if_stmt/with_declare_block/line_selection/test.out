main.adb(<IfStmt main.adb:5:11-11:17>) - 5:1-11:17
^
      if Item.kind = Snippet then
         declare
            X : Integer;
         begin
            null;
         end;
      end if;$

